ControlRoomApplication/ControlRoomApplication/GUI/RTControlForm.cs

```diff
@@ -519,34 +519,67 @@ private async void runControlScript_Click(object sender, EventArgs e)
                         string input = "";
+                        string prompt = "";
                         string[] values;
-                        Entities.Orientation currentOrientation = rtController.GetCurrentOrientation();
+                        bool invalidInput = false;
 
-                        // Get validated user input for azimuth position
-                        do
+                        // Determine limits based off software stops checkbox
+                        if (SoftwareStopsCheckBox.Checked)
+                        {
+                            // Set to software stop limits used by telescope
+                            elevationHighLimit = MiscellaneousConstants.MAX_SOFTWARE_STOP_EL_DEGREES;
+                            elevationLowLimit = MiscellaneousConstants.MIN_SOFTWARE_STOP_EL_DEGREES;
+                        }
+                        else
                         {
-                            input = Interaction.InputBox("The Radio Telescope is currently set to be type " + rtController.RadioTelescope.teleType + "." +
+                            elevationHighLimit = SimulationConstants.LIMIT_HIGH_EL_DEGREES;
+                            elevationLowLimit = SimulationConstants.LIMIT_LOW_EL_DEGREES;
+                        }
+
+                        prompt = "The Radio Telescope is currently set to be type " + rtController.RadioTelescope.teleType + "." +
                             " This script is best run with a telescope type of SLIP_RING.\n\n" +
                             "Please type an a custom orientation containing azimuth between 0 and 360 degrees," +
-                                " and elevation between " + Constants.SimulationConstants.LIMIT_LOW_EL_DEGREES + " and " + Constants.SimulationConstants.LIMIT_HIGH_EL_DEGREES +
+                                " and elevation between " + elevationLowLimit + " and " + elevationHighLimit +
                                 " degrees. Format the entry as a comma-separated list in the format " +
-                                "azimuth, elevation. Ex: 55,80",
-                                "Azimuth Orientation", currentOrientation.Azimuth.ToString() + "," + currentOrientation.Elevation.ToString());
+                                "azimuth, elevation. Ex: 55,80\n";
+                        
+                        Entities.Orientation currentOrientation = rtController.GetCurrentOrientation();
+
+                        // Get validated user input for azimuth position
+                        do
+                        {
+                            if (invalidInput)
+                            {
+                                azimuthPos = 0;
+                                elevationPos = 0;
+
+                                input = Interaction.InputBox(prompt + " === INVALID VALUES ENTERED === ", "Azimuth Orientation", currentOrientation.Azimuth.ToString() + "," + currentOrientation.Elevation.ToString());
```

Comment:
Show Todd the error output and make sure that he is fine with the way it looks. Basically, any time something on the UI gets changed, run it by him even if it is something small like this.

- [ ] Run UI change past Todd